src/components/totals/FundsRaised/__tests__/FundsRaised-test.js

```diff
@@ -255,4 +256,26 @@ describe('FundsRaised', function() {
     });
   });
 
+  describe('Callback function', function() {
+    beforeEach(function() {
+      jest.dontMock('../../../../api/totals');
+      sinon.stub(totals, 'findByCharities', function(params, callback) {
+        callback({
+          total_amount_cents: {
+            sum: 10000
+          }
+        })
+      });
+    });
+
+    afterEach(function() {
+      totals.findByCharities.restore();
+    })
+
+    it('calls a callback function after data is fetched', function() {
+      var cb = jest.genMockFunction();
+      var element = TestUtils.renderIntoDocument(<FundsRaised charityUid="au-24" onLoad={ function(res) { cb(res) } } />);
+      expect(cb.mock.calls.length).toBe(1);
```

Comment:
You could use your return value here to assert something about the param to the callback?